src/front/spv/function.rs

```diff
@@ -189,7 +189,7 @@ impl<I: Iterator<Item = u32>> super::Parser<I> {
         if let Some(ep) = self.lookup_entry_point.remove(&fun_id) {
             // create a wrapping function
             let mut function = crate::Function {
-                name: None,
+                name: Some(ep.name.clone()),
```

Comment:
@kvark How about this fix ?

Reply:
this seems fine to me, but we can probably do better, like `Some(format!("{}Wrap", ep.name))`. Otherwise, we'll have both the entry point and the function wrapper to have the same name.